setup-disk.in

```diff
@@ -659,8 +659,10 @@ setup_partitions() {
 	# create new partitions
 	(
 		for line in "$@"; do
-			echo "$start,$line"
-			start=
+			if case $line in 0M*) false;; *) true;; esac; then
```

Comment:
`if case ...;; esac; then` seems a bit confusing. Wouldn't it be clearer with something like:
````sh
    case $line in
         0M) ;;
         *)  echo "$start,$line"; start=  ;;
    esac
````

Reply:
Perhaps, but I figured that given it is a single-purpose case statement that it'd be fine on one line.

The only reason I'm using the case statement in this way is to ensure maximum shell compatibility, as otherwise I would be resorting to "bashisms".

If you wish for it to be structured that way, I have no opposition, but it may encourage extending this section for more "configuration options" later by other individuals who misinterpret its original intention.